test/operators.jl

```diff
@@ -339,3 +339,11 @@ end
     Base.:(<)(::B46327, ::B46327) = false
     @test B46327() <= B46327()
 end
+
+@testset "getproperty/hasproperty" begin
+    gp = Base.Fix2(Base.getproperty, :x)
+    @test Base.getproperty(:x) == gp
+    @test Base.hasproperty(:x) == Base.Fix2(Base.hasproperty, :x)
+    @test Base.getproperty(:x) == gp
```

Comment:
Duplicated test case.

Reply:
Sorry, this was supposed to be `Base.getproperty(:x)(gp) == :x` so it could test that `getproperty` returned the correct field.

Reply:
That seems a bit confusing honestly, and potentially fragile to future changes to the `Fix2` internal. I think it's better to define a specific struct for this test set whose field name isn't used anywhere else.

Reply:
Sorry, that was a poor decision on my part. I've replaced it with a `NamedTuple`

Reply:
@aviatesk, are the changes for these tests satisfactory?